syntax/neomake/qf.vim

```diff
@@ -0,0 +1,5 @@
+syntax clear
```

Comment:
I suppose it would.  But, as stated above, this would ideally run *only* for Neomake.  For Neomake-specific syntax, custom configs would live in `&rtp/syntax/neomake/qf.vim`.

The reason I did it like this is so that it wouldn't clash with any existing `syntax/qf.vim` configs, as well as allow maker-specific highlighting like in the example for Python.